lib/python/pyflyby/_dynimp.py

```diff
@@ -111,8 +111,9 @@ def _add_import(ip, names: str, code: str) -> None:
     private version of add_import
     """
     assert ip is not None
-    mang = PYFLYBY_LAZY_LOAD_PREFIX + names.replace(",", "_").replace(" ", "_")
-    a: FrozenSet[Import] = ImportSet(f"{mang} import {names}")._importset
+    module = PYFLYBY_LAZY_LOAD_PREFIX.split(" ")[1]
```

Comment:
```suggestion
    module = PYFLYBY_LAZY_LOAD_PREFIX.split()[1]
```
No need to pass `" "`